lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -301,12 +301,12 @@
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="482"/>
       <source>Warning</source>
-      <translation type="unfinished">Warning</translation>
+      <translation>Avertissement</translation>
```

Comment:
There is (at least) another translation of the exact same sentence above. I very much prefer this one.